redwood-treehouse-composeui/build.gradle

```diff
@@ -1,16 +1,32 @@
 apply plugin: 'com.android.library'
-apply plugin: 'org.jetbrains.kotlin.android'
+apply plugin: 'org.jetbrains.kotlin.multiplatform'
+apply plugin: 'app.cash.redwood.build.compose'
 
-dependencies {
-  api projects.redwoodTreehouse
-  implementation libs.androidx.compose.foundation
-  implementation projects.redwoodWidgetCompose
+kotlin {
+  android {
+    publishLibraryVariants('release')
+  }
+
+  iosArm64()
+  iosX64()
+  iosSimulatorArm64()
+
+  jvm()
+
+  macosArm64()
+  macosX64()
```

Comment:
Convention plugins sorely needed. This is everything but JS.

On my TODO list...